app/controllers/items_controller.rb

```diff
@@ -17,13 +18,18 @@ def show
   end
 
   def new
-    @item = Item.new(status: 'Available')
+    @item_type = ItemType.find ( params[:item_type_id])
+    @item = Item.new(
+            status: 'Available',
+            item_type_id: params[:item_type_id])
+
     @page_title = "New Item"
   end
 
   def edit
     @item = Item.find(params[:id])
```

Comment:
add the `.includes(:item_type)` to save the 2nd query happening on the next line.